packages/db/db.mjs

```diff
@@ -38,6 +39,7 @@ program.register('seed', seed)
 program.register('types', generateTypes)
 program.register('schema graphql', printGraphQLSchema)
 program.register('schema openapi', printOpenAPISchema)
+program.register('generate migration', generateMigration)
```

Comment:
I'm not good in command naming, but I would move it under `migrate` command namespace. Something like `platformatic db migrate next`, but it also a bad example because it might mean "apply next migration".

@mcollina @simonplend Any thoughts about good name for this command?

Reply:
I agree that this should be under the `db migrate` command namespace, perhaps as `db migrate create`?

Sidenote: I think the namespace for migrations should be renamed to `db migrations`, but that's a separate discussion.

Reply:
+1 for `db migrate create`

Reply:
@stanimirovv Can you update command to `db migrate create`?

Reply:
Yep, understood.

Reply:
@stanimirovv Please update the command name.

As a good rule, please don't resolve conversations in your own PR. This will make it easier to review your PR.